mutacc/resources/__init__.py

```diff
@@ -1,27 +1,22 @@
-import pkg_resources
+from importlib_resources import files
 
-case_filename = "resources/case.yaml"
-father_fastq1_filename = "resources/father_R1.fastq.gz"
-father_fastq2_filename = "resources/father_R2.fastq.gz"
-father_bam_filename = "resources/father.bam"
-mother_bam_filename = "resources/mother.bam"
-child_bam_filename = "resources/child.bam"
-vcf_filename = "resources/variant1.vcf.gz"
+BASE_PATH = "mutacc.resources"
 
-vcf_parser_file = "resources/vcf-info-def.yaml"
+case_filename = "case.yaml"
+father_fastq1_filename = "father_R1.fastq.gz"
+father_fastq2_filename = "father_R2.fastq.gz"
+father_bam_filename = "father.bam"
+mother_bam_filename = "mother.bam"
+child_bam_filename = "child.bam"
+vcf_filename = "variant1.vcf.gz"
+vcf_parser_file = "vcf-info-def.yaml"
 
-path_to_case_file = pkg_resources.resource_filename("mutacc", case_filename)
-path_to_background_fastq1_file = pkg_resources.resource_filename(
-    "mutacc", father_fastq1_filename
-)
-path_to_background_fastq2_file = pkg_resources.resource_filename(
-    "mutacc", father_fastq2_filename
-)
-path_to_background_bam_file = pkg_resources.resource_filename(
-    "mutacc", father_bam_filename
-)
 
-default_vcf_parser = pkg_resources.resource_filename("mutacc", vcf_parser_file)
+path_to_case_file = str(files(BASE_PATH).joinpath(case_filename))
```

Comment:
Use pathlib instead?

Reply:
importlib.resources is a context manager for pathlib that basically gives you the pathlib.Path. 
I've followed the instructions given by the [pkg_resources migration guide] (https://importlib-resources.readthedocs.io/en/latest/migration.html). And we use importlib_resources anyway when retrieving the software version so why not also here?